setup.py

```diff
@@ -8,9 +8,9 @@
 __version__ = "0.6.0"
 
 inst_reqs = [
-    f"titiler.core=={__version__}",
-    f"titiler.mosaic=={__version__}",
-    f"titiler.application=={__version__}",
+    f"titiler.core @ file://localhost/{os.getcwd()}/src/titiler/core",
+    f"titiler.mosaic @ file://localhost/{os.getcwd()}/src/titiler/mosaic",
+    f"titiler.application @ file://localhost/{os.getcwd()}/src/titiler/application",
```

Comment:
This uses the three TiTiler modules (`core`, `mosaic`, and `application`) from the local filesystem rather than installing the mainstream releases from pip.

Reply:
I wish pip had better documentation on this. Poetry makes it really easy in contrast.